### DISCUSSION TITLE
Can NiceGUI use the latest version of ECharts

### DISCUSSION THREAD

**Discussion by CrystalWindSnake** [zauberzeug/nicegui#2933] [Q&A]
### Question

ECharts has now released version 5.5. Can NiceGUI use the latest version of ECharts in the next release?

**Comment by falkoschindler** [comment]
We usually update third-party libraries with every major NiceGUI release, i.e. 1.1, 1.2, 1.3, 1.4 and the upcoming 2.0. But we could update individual libraries out of schedule. Is there a specific reason you need ECharts 5.5?

**Reply by CrystalWindSnake** [reply]
Alright, I understand. I think it's better to stick to your plan for the update.
I mainly want to use its [drill-down animation feature](https://github.com/apache/echarts/pull/17611) . I can replace the echarts.js file by myself.
